auth/guard.go

```diff
@@ -80,3 +81,9 @@ func jwtSigningMethod() jwt.SigningMethod {
 		return nil
 	}
 }
+
+// extractJWTFromHeader will parse the header string and return just the token
+// header should be formatted as Bearer [TOKEN]
+func extractJWTFromHeader(header string) string {
+	return header[7:]
```

Comment:
It should never get here if the header doesn't match the regex from `validateAuthHeader`